### PR TITLE
Fix #13044: Lyrics/Lyricist

### LINKED ISSUE #13044: [MU4 Issue] Lyrics defaulting to Lyricist Text Style
**Describe the bug**
Lyrics in MuseScore 4 are reportedly using the Lyricist style, instead of the Lyrics Odd/Even Lines styles. 

**To Reproduce**
1. Create a new score and add some notes
2. Add lyrics
3. Select a syllable, and click More... under Properties to view the Text Style

**Expected behavior**
The Text Style should be Lyrics Odd/Even Lines, depending on which verse the selected syllable is from. 

**Screenshots**
image

**Platform information**
 - OS: Windows 10

### PR DESCRIPTION
**Pull request by Wim4rk**
Resolves: *13044*

*Corrected simple typo lyrics/lyricist*

- [ x] I signed [CLA](https://musescore.org/en/cla)
- [ x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [ x] I made sure the code compiles on my machine
- [ x] I made sure there are no unnecessary changes in the code
- [ x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [ x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [ x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Review by shoogle — CHANGES_REQUESTED** [review]
Welcome! Thanks for the fix!

Please remove all the merge commits from this branch. You can do this via an interactive rebase (look for tutorials online) but an easier method is probably to delete the branch and then try again following the instructions in:

- https://github.com/musescore/MuseScore/wiki/Submit-a-Pull-Request
- https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/git-workflow

It is **not** necessary to close this pull request. Just give the new branch the same name as this one (`lyrics_text`) and then force push to update the PR:

```Bash
git push -f origin lyrics_text
```

File: src/appshell/qml/Preferences/internal/ProgrammeStartSection.qml [hunk 1]
```diff
@@ -27,7 +27,7 @@ import MuseScore.UiComponents 1.0
 BaseSection {
     id: root
 
-    title: qsTrc("appshell/preferences", "Program start")
+    title: qsTrc("appshell/preferences", "Programme start")
```
Comment (shoogle) [inline src/appshell/qml/Preferences/internal/ProgrammeStartSection.qml:30]: "Program" was correct. We use American English in all strings, but even in British English it is normal to use the American spelling when referring to computer programs.

Comment (Wim4rk) [reply src/appshell/qml/Preferences/internal/ProgrammeStartSection.qml:30]: Hmmm... I tried to fix 13044 - Lyrics default to lyricist text-style. This is an old (rejected) PR...
Would you mind killing this PR, I'll make a new one?

Comment (shoogle) [reply src/appshell/qml/Preferences/internal/ProgrammeStartSection.qml:30]: It's possible to drop the other changes and fix only the lyricist bug in this PR, but feel free to close it and make a new one if you prefer.

**Comment by Jojo-Schmitz** [comment]
Any replacement PR? That last commit does look right to me

**Comment by Wim4rk** [comment]
Comming, I managed to wreck my build (Qt-issues), and I'd like to try it again.

**Comment by Wim4rk** [comment]
And it's up (running checks).

**Pull request closed without merging**